<?xml version="1.0"?>
<launch>
  <node name="plan2ctrl" pkg="motion_planner" type="motion_planner_hhri" output="screen" respawn="false">
	  <param name="action_mode" value="vision" />
	  <param name="wait_for_force" value="true" />
	  <param name="simulation" value="false" />
  	  <param name="world_frame" value="/world_frame" />
	  <param name="model_dt" value="0.01" />
	  <param name="reachingThreshold" value="0.02" />
	  <param name="orientationThreshold" value="0.02" />
	  <param name="use_boxy_tool" value="false" />
	  <param name="autonomous" value="true" />
	  <param name="model_base_path" value="$(find motion_planner)../epfl-task-models/DoughRollingModels" />
	  <!-- param name="path_matlab_plot" value="/home/nbfigueroa/catkin_ws/src/lasa-dynamics-plotter/epfl-task-models-plot/DoughRollingModels/" /-->
  </node>
</launch>
